prototype/app/views/index.html

```diff
@@ -3,33 +3,101 @@
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
 
-{% set pageName="Home" %}
-
-{% block pageTitle %}Home{% endblock %}
+{% set pageTitle = "" %}
 
 {% block content %}
+<style>
 
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-full">
-    <h1 class="govuk-heading-l">Consultations frontend prototype</h1>
-
-    <h2 class="govuk-heading-m">Demo consultation</h2>
-    <p>The demo consultation shows how each page will look and what features will be available.</p>
-    {{ govukButton({
-      text: "View consultation",
-      href: "/demo-consultation",
-      isStartButton: true
-    }) }}
-
-    <h2 class="govuk-heading-m govuk-!-margin-top-3">Upload consultation</h2>
-    <p>This is an MVP version just for use by the i.AI team currently. Longer term this can be expanded to be useable by others. At which point we'll need to consider what additional data/instructions will be required.</p>
-    {{ govukButton({
-      text: "Upload consultation",
-      classes: "govuk-button--secondary",
-      href: "/upload"
-    }) }}
+  .govuk-main-wrapper {
+    padding-top: 0;
+  }
+  .govuk-phase-banner {
+    border-bottom: 0;
+  }
+  .govuk-phase-banner__text .govuk-link {
+    /*color: white;*/
+  }
+</style>
+<div class="x-govuk-masthead x-govuk-masthead--pink">
+  <div class="govuk-width-container">
+    {#
+    <div class="govuk-phase-banner x-govuk-phase-banner--inverse">
+      <p class="govuk-phase-banner__content">
+        <strong class="govuk-tag govuk-phase-banner__content__tag x-govuk-tag--inverse">
+          Prototype
+        </strong>
+        <span class="govuk-phase-banner__text">
+          This is a new service – your <a class="govuk-link" href="#feedback">feedback</a> will help us to improve it.
+        </span>
+      </p>
+    </div>
+    #}
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds-from-desktop">
+        <h1 class="x-govuk-masthead__title">
+          Take the responses to your consultation and use machine learning to create themes
```

Comment:
I'd like to finish the conversation on copy (in the GDoc) before we go in on this!

Reply:
I can see it's now updated in the doc. I've just push a commit to change it to:
"Use machine learning to help you understand consultation responses"
(this definitely reads better to me)